test/generic/Matrix-test.jl

```diff
@@ -403,6 +403,16 @@ end
 
    @test issquare(A)
 
+   @test A[1:end, 1:end] == A
+   @test firstindex(A, 1) == 1
+   @test lastindex(A, 1) == nrows(A)
+   @test lastindex(A, 2) == ncols(A)
+   @test_throws ErrorException lastindex(A, 3)
+
+   if VERSION >= v"1.4"
+      @test A[begin:end, :] == A
```

Comment:
Actually, the version check is not enough to make this test, as in older Julia versions, `begin` within indexing triggers a warning. Even with a `@static if` it seems to fail, so probably this test just has to be removed :shrug:

Reply:
Who knows. I removed it.